packages/near-membrane-base/src/__tests__/environment.spec.ts

```diff
@@ -36,6 +36,7 @@ describe('VirtualEnvironment', () => {
                     return v;
                 },
             });
+            ve.link('globalThis');
```

Comment:
a `VirtualEnvironment` makes no assumptions anymore about the reflection properties of the environment.